pom.xml

```diff
@@ -64,7 +64,7 @@
     <properties>
         <revision>2.23</revision>
         <changelist>-SNAPSHOT</changelist>
-        <jenkins.version>2.121.1</jenkins.version>
+        <jenkins.version>2.145</jenkins.version>
```

Comment:
That sounds like a great idea - didn't think of it yet. However I'm not sure how then to run those tests against 2.145+? Thought that the test harness will always pick the Jenkins version as given in the pom.xml?

Reply:
Take a look at the possible parameters for `buildPlugin` [here](https://github.com/jenkins-infra/pipeline-library#optional-arguments). If you have `buildPlugin(jenkinsVersions: [null, '2.145'])`, then the maven commands run will be something like the following:

    mvn clean install 
    mvn clean install -Djenkins.version=2.145

The property in the second version will overrride the value in the pom, allowing you test against a different Jenkins version.

Reply:
Thanks for the insight. Works as described.
I just pushed my latest changes after merging in the official master branch.
However for merging my changes to the Jenkinsfile I'll need your help.

Reply:
Btw.: Are there any plans to support changes to the Jenkinsfile like - let's say - in case there are changes that there will be two builds: One build as currently with the unchanged Jenkinsfile and the other one incorporating the changes?